src/section-solving-system-equations-by-graphing.mbx

```diff
@@ -317,76 +317,115 @@
+                                    legend cell align=left, legend pos=north west,
+                                    ]
+                            \addplot[primarycurve,domain = -1.5:5.5]{2*x-4};
+                            \addplot[secondarycurve,domain = -4:3]{2*x+1};
+                            \addplot[soliddot,color=firstcolor] coordinates {(0,-4) (1,-2) (2,0)};
+                            \addplot[soliddot,color=secondcolor] coordinates {(0,1) (1,3) (2,5)};
+                            \legend{$y=2x-4$,$y=2x+1$}
+                        \end{axis}
+                    \end{tikzpicture}]]>
+                    </latex-image-code>
+                </image>
+            </figure>
+
+            For this system of equations, what is the solution? We know that since the two lines have the same slope (and are thus parallel) they will never intersect. This means that there is <em>no solution</em> to this system of equations. We'd write the solution set as <m>\emptyset</m>. 
+            </p>
+        </example>
 
-        <p>We can see those two lines are parallel, thus will never intersect. The system has no solution.</p>
-
-        <p>Recall that two lines are parallel if they have the same slope. Next time, when we solve a system of equations with the same slope, we can simply conclude without graphing that the system has no solution.</p>
-
-        <p>Next, let's solve the system<md>
-            <mrow>
-                \left\{
-                    \begin{alignedat}{4}
-                    y \amp {}={} \amp 2x-4 \\
-                    2x-y \amp {}={} \amp 4 \\
-                    \end{alignedat}
-                \right.
-            </mrow></md></p>
-
-        <p>We will change the equation <m>2x-y=4</m> into slope-intercept form:<md>
-            <mrow>2x-y\amp=4</mrow>
-            <mrow>2x-y\subtractright{2x}\amp=4\subtractright{2x}</mrow>
-            <mrow>-y\amp=-2x+4</mrow>
-            <mrow>(-1)\cdot(-y)\amp=(-1)\cdot(-2x+4)</mrow>
-            <mrow>y\amp=2x-4</mrow>
-        </md></p>
-
-        <p>It turned out that those two equations are equivalent. Since they are the same line (or they overlap each other), any point on them is an intersection. We say the system has infinitely many solutions.</p>
-
-        <p>Let's summarize our learning:<ul>
-            <li><p>Parallel Lines: If linear equations in a system have the same slope with different <m>y</m>-intercept, the system has no solution.</p></li>
-            <li><p>Overlapping Lines: If linear equations in a system have the same slope and the same <m>y</m>-intercept, the system has infinitely many solutions.</p></li>
-            <li><p>Intersecting Lines: If linear equations in a system have different slopes, the system has one solution.</p></li>
+        <example xml:id="example-solving-systems-by-grahping-equivalent-equations">
+            <p>Next we'll look at a slightly different special case. Let's start with this system of equations:
```

Comment:
Here is a `p` that has a `figure` inside it, which is a DTD violation. `figure` is a peer of `p`. I think I would structure this as `p`, `p`, `figure`, `p`.